css/components/Grid.css

```diff
@@ -0,0 +1,26 @@
+@import "../style.css";
+
+.parent-grid{
+    margin: 1rem;
+    grid-row-gap: 1rem;
+}
+
+.grid{
+    font-size: 1rem;
+    width: max-content;
+    grid-column-gap: 1rem;
+}
+
+.grid-2{
+    grid-template-columns: 10rem 10rem;
+}
+
+.grid-3{
+    grid-template-columns: 10rem 10rem 10rem;
+}
```

Comment:
Suggestion: repeat(2,10rem) and repeat(3,10rem) can be used

Reply:
done